modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/StyleBookField.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+export default function StyleBookField({children, errors, id, label, name}) {
+	const errorMessage = errors[name];
+
+	return (
+		<ClayForm.Group className={errorMessage ? 'has-error' : null}>
```

Comment:
Instead of creating this component, you should be able to use https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx#L1